build.sbt

```diff
@@ -33,15 +33,16 @@ val zioVersion = "2.0.15"
 lazy val root = project
   .in(file("."))
   .settings(
-    skip / publish := true,
+    publish / skip := true,
     unusedCompileDependenciesFilter -= moduleFilter("org.scala-js", "scalajs-library")
   )
   .aggregate(
     zioCliJVM,
     zioCliJS,
     examplesJVM,
     examplesJS,
-    docs
+    docs,
+    sbtZioCli,
```

Comment:
This makes `sbt publish` or `sbt publishLocal` include the project `sbtZioCli` by default, which it currently doesn't.